docs/docs/12_task_queues_emails/01_send_emails_python_mailgun/end/migrations/env.py

```diff
@@ -81,8 +80,7 @@ def process_revision_directives(context, revision, directives):
             connection=connection,
             target_metadata=target_metadata,
             process_revision_directives=process_revision_directives,
-            compare_type=True,
-            **current_app.extensions['migrate'].configure_args
+                **current_app.extensions['migrate'].configure_args
```

Comment:
Is the indentation correct here?

```suggestion
            **current_app.extensions['migrate'].configure_args
```